examples/mnist/train_lenet_docker.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+set -e
+# The following example allows for the MNIST example (using LeNet) to be run
+# using the caffe docker image instead of building from source.
+# The GPU-enabled version of Caffe can be used, assuming that nvidia-docker
+# is installed, and the GPU-enabled Caffe image has been built.
+# Setting the GPU environment variable to 1 will enable the use of nvidia-docker.
+# e.g.
+#   GPU=1 ./examples/mnist/train_lenet_docker.sh
+#
+# Not the use of the -u, -v, and -w command line options to ensure that
+# files are created under the ownership of the current user, and in the
+# current directory.
+
+
+
+if [ x"$GPU" = x"1" ]
+then
+DOCKER_CMD=nvidia-docker
+IMAGE=caffe:gpu
+else
+DOCKER_CMD=docker
+IMAGE=caffe:cpu
+fi
+
+echo "Using $DOCKER_CMD to launch $IMAGE"
+
+$DOCKER_CMD run --rm -ti \
+    -u $(id -u):$(id -g) \
+    -v $(pwd):/workspace \
+    -w /workspace \
+    $IMAGE \
+    caffe train --solver=examples/mnist/lenet_solver.prototxt $*
```

Comment:
Won't this fail since the MNIST data hasn't been downloaded nor have the lmdb been created?